assets/javascripts/job_next_previous.js

```diff
@@ -86,18 +86,22 @@ function renderJobResults(data, type, row) {
         if (count++) {
             var more = row.failedmodules.length - count + 1;
             if (more > 0 && limit < 12) {
-                html += '+' + more;
+                html += '<a title="';
```

Comment:
I've just wanted to click the merge button but I think there's still one mistake. This should be a span and not a link because there's no href or name attribute. The same counts for the server-side rendered version.

Reply:
Replaced `<a>` with `<span>`.